tasks/install_repo.yml

```diff
@@ -8,7 +8,6 @@
 # therefore handler notifications are not required at this step.
 
 - name: ClickHouse Apt repository
-  check_mode: false
```

Comment:
But this will break `--check` for regular users?

Reply:
Oh, I see, and now we will have true --check mode, so we can remove `check_mode: false` for `update_cache` and also remove `molecule-idempotence-notest` right?

Reply:
I thought update cache left intentionally -- it doesn't modify the system (at least anything related to the role tasks), and allows to show that latest version of package should be updated.

But will check, maybe it'll be the right thing to do.

Reply:
The reason that `update_cache` runs even with `--check` because `apt install` fails, and this is the only reason

But yes, if the repos already configured, then it will also check that such version is available, and I like this, but not sure that it worth it. Anyway I'm OK with both

Reply:
Let's remove it then in order not to perform any actions at all.